aries_cloudagent/issuer/indy.py

```diff
@@ -246,26 +246,46 @@ async def revoke_credentials(
             cred_revoc_ids: sequences of credential indexes in the revocation registry
 
         Returns:
-            the combined revocation delta
+            Tuple with the combined revocation delta, list of cred rev ids not revoked
 
         """
+        failed_crids = []
         tails_reader_handle = await create_tails_reader(tails_file_path)
 
         result_json = None
         for cred_revoc_id in cred_revoc_ids:
             with IndyErrorHandler("Exception when revoking credential", IssuerError):
-                # may throw AnoncredsInvalidUserRevocId if using ISSUANCE_ON_DEMAND
-                delta_json = await indy.anoncreds.issuer_revoke_credential(
-                    self.wallet.handle, tails_reader_handle, revoc_reg_id, cred_revoc_id
-                )
+                try:
+                    delta_json = await indy.anoncreds.issuer_revoke_credential(
+                        self.wallet.handle,
+                        tails_reader_handle,
+                        revoc_reg_id,
+                        cred_revoc_id,
+                    )
+                except IndyError as error:
```

Comment:
Nice, this is great.

Also note @esune I believe this should cure the problem you were seeing without any changes on your end.

Reply:
... then take the troublesome cred_rev_ids on the rev_reg_id and purge them via clear_pending_revocations API as per issue_credentials/v1_0/routes.py, submitting body
```
{
    "rrid2crid": {
        "<rev-reg-id>": ["<cred-rev-id>", "<cred-rev-id>", ...]
    }
}
```